changelog/1488.doc.2.rst

```diff
@@ -0,0 +1 @@
+Set minimum version of `sphinx` to ``v4.4``".
```

Comment:
I thought I fixed that!  I caught myself doing that earlier. 🤦‍♂️

Reply:
Not a problem whatsoever — it was kind of you to leave something for me to suggest during code review, since that helps me feel like I'm being productive and useful!